src/shared/services/coordinate-reference-system/convertCoordinates.ts

```diff
@@ -0,0 +1,38 @@
+import { rdToWgs84, wgs84ToRd } from './crs-converter'
+
+type Coordinate = [number, number]
+
+function convertCoordinates(location: Coordinate, type: 'RD' | 'WGS84') {
+  let wgs84Location
+  let rdLocation
```

Comment:
Instead of defining these values here and giving them a value later on, why not initialize them here directly?

```suggestion
  const wgs84Location = type === 'WGS84' ? location : rdToWgs84({
    x: rdLocation[0],
    y: rdLocation[1],
  })

  const rdLocation = type === 'RD' ? location : wgs84ToRd({
    lat: wgs84Location[0],
    lng: wgs84Location[1],
  })
```

Reply:
Nice one, was brainlessly copy pasting from old angular junk